libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxManeuverView.kt

```diff
@@ -171,6 +171,30 @@ class MapboxManeuverView : ConstraintLayout {
         TextViewCompat.setTextAppearance(mainLayoutBinding.stepDistance, style)
     }
 
+    /**
+     * Allows you to change the text appearance of primary maneuver text in upcoming maneuver list.
+     * @param style Int
+     */
+    fun updateUpcomingPrimaryManeuverTextAppearance(@StyleRes style: Int) {
```

Comment:
How should the style res look like to be able to change the colors? How do we know which values to override? Could you add an example and reference in the docs? Or at least link to `TextViewCompat.setTextAppearance` so that developers know what they need to provide without looking at the source code.

Same for all the other methods in the class.

Reply:
Done.